kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/UIServicesClientTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2019 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.server.client;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+import org.kie.server.api.KieServerConstants;
+import org.kie.server.api.rest.RestURI;
+import static org.kie.server.api.rest.RestURI.CONTAINER_ID;
+import static org.kie.server.api.rest.RestURI.FORM_URI;
+import static org.kie.server.api.rest.RestURI.TASK_FORM_GET_URI;
+import static org.kie.server.api.rest.RestURI.build;
+
+public class UIServicesClientTest extends BaseKieServicesClientTest {
+
+    private final String dummyFormContent = "form content";
+
+    private UIServicesClient uiClient;
+
+    @Before
+    public void createClient() {
+
+        config.setCapabilities(List.of(KieServerConstants.CAPABILITY_BPM_UI));
```

Comment:
Our code needs to be compatible with Java 8. List.of() was brought in Java 9. Can you please refactor the code to be Java 8 compatible?

Reply:
Of course! I wasn't aware of the Java 8 compatibility, sorry.
Will to that right away.

Reply:
Converted to Arrays.asList on last commit.